cellpose/io.py

```diff
@@ -560,6 +561,21 @@ def masks_flows_to_seg(images, masks, flows, file_names, diams=30., channels=Non
 
     np.save(base + "_seg.npy", dat)
 
+def save_features_csv(file_name):
+    """ save features to .csv file and remove if it already exists
```

Comment:
Really, really tiny nitpick, but I would put a line break after the quotation marks for consistency